how-to-take-notes.md

```diff
@@ -57,22 +57,22 @@ The notes may or may not be initialised yet. If they are not, you can use this t
 -------------------------------------------------------------------------------------------------------------------------------------------
 Template:
 -------------------------------------------------------------------------------------------------------------------------------------------
-    
-    
+
+
 ## X.Y.Z Agenda Item 
- 
+
 (Presenter Full Name)
- 
- - [proposal](...link to proposal...)
- - [slides](...link to slides...)
- 
+
+ - [proposal](...link to proposal...)
+ - [slides](...link to slides...)
+
 JB: Ja ja ja....
- 
+
 MKR: Ne! Ne! Ne!
- 
+
 #### Conclusion/Resolution
- 
-- Stage -1 
+
+- Stage 0
```

Comment:
As long as we're tweaking this, might as well set a better example:

```suggestion
- Proposal moves to stage 1
```

(or something)

Reply:
whoops, forgot to update this prior to merge.  this doc [needs a lot of work](https://github.com/tc39/how-we-work/issues/143) in general though...